src/components/App-footer/app-footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { useStyles } from './app-footer.styles';
+
+import FooterLists from '../../containers/footer-lists';
+import FooterLinks from '../../containers/footer-links';
+
+const AppFooter = () => {
+  const classes = useStyles();
```

Comment:
Please, use `styles` instead of `classes`

Reply:
Resolved